lib/openid_connect.ex

```diff
@@ -266,6 +283,43 @@ defmodule OpenIDConnect do
     end
   end
 
+  defp verify_claims(claims, config) do
+    with :ok <- verify_exp_claim(claims, exp_leeway(config)),
+         :ok <- verify_aud_claim(claims, client_id(config)) do
```

Comment:
Is the `client_id` always going be the same as the expected `aud`? I was under the impression these could be different. Apologies in advance if this comment betrays my ignorance :).

Reply:
From https://openid.net/specs/openid-connect-core-1_0.html#IDToken (emphasis mine)

> aud
> REQUIRED. Audience(s) that this ID Token is intended for. **_It MUST contain the OAuth 2.0 client_id of the Relying Party as an audience value_**. It MAY also contain identifiers for other audiences. In the general case, the aud value is an array of case sensitive strings. In the common special case when there is one audience, the aud value MAY be a single case sensitive string. 

The `audience_matches?` private helper function handles the special case when the `aud` claim may contain multiple audiences.